src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -124,7 +124,14 @@ def create(dataset, target, feature = None, model = 'resnet-50',
     # Check model parameter
     allowed_models = list(_pre_trained_models.MODELS.keys())
     if _mac_ver() >= (10,14):
-        allowed_models.append('VisionFeaturePrint_Screen')
+        allowed_models.append('VisionFeaturePrint_Scene')
+
+        # Also, to make sure existing code doesn't break, replace incorrect name
+        # with the correct name version
+        if model == "VisionFeaturePrint_Screen":
+            print "WARNING: Correct spelling of model name is VisionFeaturePrint_Scene."
```

Comment:
This will fail in python 3. Need parentheses. 

Something about this argument value being removed in a future release would be good too.

Reply:
Great catch!